src-gui/src/renderer/components/other/ClickableAddress.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+import { Box, Button, Tooltip, Typography, WithStyles } from "@material-ui/core";
+import { FileCopyOutlined } from "@material-ui/icons";
+import { writeText } from "@tauri-apps/plugin-clipboard-manager";
+import InlineCode from "./InlineCode";
+
+type Props = {
+    address: string,
+    noIcon?: boolean
+}
+
+/** Display addresses monospaced and clickable such that a click copies the address to the clipboard. */
+export default function ClickableAddress({ address, noIcon = false }: Props) {
+    // Signal that the address was copied
+    const [copied, setCopied] = useState(false);
+    const tooltip = copied ? "copied" : "copy";
```

Comment:
I'd make the text here more descriptive. Something like:
"Copied to clipboard" : "Click to copy"